dev-keys.yml

```diff
@@ -87,3 +87,62 @@ keys:
       V7ZuRp//4vIzEONqqyLUFt1JZKuCwWQB2sRjKWpqRB07C0o243XefD5f7844nmXq
       H0xKMFRYjDqhFdTXFWVPnk/tu/MxoSTdK0FfsgUnjs+TGvN5TnBW
       -----END RSA PRIVATE KEY-----
+  df88fdad2612ae1e80571120e6c6371f55896696:
+    platform: sdc
+    purpose: supplementary_data
+    service: sds
+    type: private
+    use: encryption
```

Comment:
Minor and non blocker, fancy adding the new attributes to the old keys. All new keys and prod keys have all these properties.